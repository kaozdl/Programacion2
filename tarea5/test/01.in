# prueba de sistema
ingresar_localidades (21,g) (15,d) (70,p) .
ingresar_localidades (61,b) (70,e) (45,n) (90,t) .
reiniciar_sistema
ingresar_localidades (21,g) (15,d) (70,p) .
ingresar_localidades (61,b) (70,e) (45,n) (90,t) .
iniciar_pedidos 10
# en cada grupo se ingresa localidad (insumo, monto) prioridad
ingresar_pedidos .
ingresar_pedidos d (1000,chapas) 4 t (500,ropa) 10 b (2000,comestibles) 3  .
ingresar_pedidos t (900,chapas) 10 t (5000,comestibles) 2 .
imprimir_totales
procesar_rango 1
procesar_rango 2
procesar_rango 3
procesar_rango 4
procesar_rango 5
reiniciar_sistema
ingresar_localidades (21,g) (15,d) (70,p) .
ingresar_localidades (61,b) (70,e) (45,n) (90,t) .
iniciar_pedidos 10
# en cada grupo se ingresa localidad (insumo, monto) prioridad
ingresar_pedidos .
ingresar_pedidos d (1000,chapas) 4 t (500,ropa) 10 b (2000,comestibles) 3  .
ingresar_pedidos t (900,chapas) 10 t (5000,comestibles) 2 .
imprimir_totales
procesar_rango 1
procesar_rango 2
procesar_rango 3
procesar_rango 4
procesar_rango 5
reiniciar_sistema
reiniciar_sistema
Fin
