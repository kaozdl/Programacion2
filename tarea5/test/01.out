1>2>3>4>
  REINICIO DE SISTEMA 
5>6>7>
 IDENTIFICADORES DE LOCALIDADES 
(1,b)(2,d)(3,e)(4,g)(5,n)(6,p)(7,t)
8>9>10>11>12>INFORME POR RANGOS
1000
0
0
2000
6400
13>d
(1000,chapas)
14>g

15>n

16>b
(2000,comestibles)
e

p

17>t
(5000,comestibles)(500,ropa)(900,chapas)
18>
  REINICIO DE SISTEMA 
19>20>21>
 IDENTIFICADORES DE LOCALIDADES 
(1,b)(2,d)(3,e)(4,g)(5,n)(6,p)(7,t)
22>23>24>25>26>INFORME POR RANGOS
1000
0
0
2000
6400
27>d
(1000,chapas)
28>g

29>n

30>b
(2000,comestibles)
e

p

31>t
(5000,comestibles)(500,ropa)(900,chapas)
32>
  REINICIO DE SISTEMA 
33>
  REINICIO DE SISTEMA 
34>Fin.
